mix.exs

```diff
@@ -91,7 +91,10 @@ defmodule Still.MixProject do
       files: [
         "LICENSE",
         "mix*",
-        "lib/*"
+        "lib/*",
+        "priv/still/*",
+        "installer/*",
```

Comment:
Do we want the installer to come bundled? Isn't it supposed to be a separate package?

Reply:
I've just realised why we want this. Faced the same issue right now 👍